l1-contracts/contracts/state-transition/chain-deps/facets/Mailbox.sol

```diff
@@ -430,6 +439,11 @@ contract MailboxFacet is ZKChainBase, IMailboxImpl, MessageVerification {
         }
     }
 
+    function pauseDepositsOnGateway(uint256 _timestamp) external onlyGatewayAssetTracker onlyGateway {
```

Comment:
nit: I think it should be a part of admin sol, since everything related to pausing deposits is already there

Reply:
yes, but doesn't fit, so moved here